### ISSUE TITLE
Add FFmpeg for video support

### ISSUE THREAD

**Issue by Kizuna-dev** [Quaver/Wobble#108]
Since Monogame doesn't support other OS for their video support, and i guess it would be a pain to handle, i think FFmpeg could be a good option, as it is supported by any platform, plus it could be easily integrated with the rate system to change the video speed etc.

**Comment by YaLTeR** [comment]
FFmpeg API is not for the faint hearted from my experience working with it. Although maybe we can take something from the osu-framework?

**Comment by Kizuna-dev** [comment]
Yeah we probably could since it's under MIT, tho i never used ffmpeg in a project, so i don't know how hard this will be to implement in wobble
